KatyaM/test_task/Server/tests/handler_test.go

```diff
@@ -0,0 +1,185 @@
+package tests
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	client2 "test_task/Server/client"
```

Comment:
```suggestion
	"test_task/Server/client"
```
Try to rename local values instead of package alias.

Reply:
Package aliases are used only if you have two packages with the same name in imports of one file